### DISCUSSION TITLE
Welcome to Oboe Discussions!

### DISCUSSION THREAD

**Discussion by philburk** [google/oboe#1131] [General]
## 👋 Welcome!
  We’re using the new GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions about using Oboe or AAudio.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rpattabi** [comment]
Hello, This is Ragu. I build apps for carnatic music students, teachers, and professionals at [Kuyil](https://kuyil.org/) along with my brother [Ananth](https://github.com/ananthp). By the way, [Carnatic music](https://en.wikipedia.org/wiki/Carnatic_music) is the classical music form from Southern India.

We use oboe in our apps. Dealing with device specific issues have been a challenge for us. Before oboe, we couldn't do much about these issues. We think we made very good progress thanks to oboe. Particularly, oboe enabled us to have deeper insight into these problems. Though challenges still remain, we feel that we now have more control over these issues.

We mainly contribute bug reports based on the analysis of the issues reported by our users. As indie audio developers, we also try to share our experiences, difficulties working with android audio eco-system, and propose ideas.
